File: 3412-permutation-difference-between-two-strings/permutation-difference-between-two-strings.java
class Solution {
    public int findPermutationDifference(String s, String t) {
        int ans = 0;
        for(int i = 0; i < s.length(); i++){
            int tm = t.indexOf(s.charAt(i));
            ans += Math.abs(i-tm);
        }
        return ans;
    }
}

File: 3412-permutation-difference-between-two-strings/permutation-difference-between-two-strings.js
/**
 * @param {string} s
 * @param {string} t
 * @return {number}
 */
var findPermutationDifference = function(s, t) {
    let ans = 0;
        for(let i = 0; i < s.length; i++){
            let tm = t.indexOf(s.charAt(i));
            ans += Math.abs(i-tm);
        }
        return ans;
};